microsoft-365/enterprise/ms-cloud-germany-transition-add-experience.md

```diff
@@ -98,31 +66,31 @@ Between Phase 2 of 9 and Phase 3 of 9, Partner Portal may not be accessible. Dur
 | Step(s) | Description | Impact |
 |:-------|:-------|:-------|
 | Remove relying party trusts from Microsoft Cloud Deutschland AD FS. | After the cut over to Azure AD is complete, the organization is fully using Office 365 services and is no longer connected to Microsoft Cloud Deutschland. At this point, the customer needs to remove the relying party trust to the Microsoft Cloud Deutschland endpoints. This can only be done when no applications of the customer point to Microsoft Cloud Deutschland endpoints when Azure AD is leveraged as an Identity Provider (IdP). | Federated Authentication organizations | None. |
```

Comment:
```suggestion
| Remove relying party trusts from Microsoft Cloud Deutschland AD FS. | After the cut-over to Azure AD is complete, the organization is fully using Office 365 services and is no longer connected to Microsoft Cloud Deutschland. At this point, the customer needs to remove the relying party trust to the Microsoft Cloud Deutschland endpoints. This can only be done when none of the customer's applications points to Microsoft Cloud Deutschland endpoints when Azure AD is leveraged as an Identity Provider (IdP). | Federated Authentication organizations | None. |
```